### DISCUSSION TITLE
Can we set savepoint(transaction) in SFDX?

### DISCUSSION THREAD

**Discussion by gtiancai** [forcedotcom/cli#2380] [Ideas]
**Is your feature request related to a problem? Please describe.**
Can we set savepoint (transaction) using SFDX?

**What are you trying to do**
eg, I wan to import data for 2 Objects with command `sfdx force:data:bulk:upsert` with 2 CSV files, and I want to rollback all data if the 2nd is failed.

**Describe the solution you'd like**
Auto-rollback all data by transaction.

[a comment by an automated account is not shown]

**Comment by gtiancai** [comment]
Any comments?

[a comment by an automated account is not shown]

**Comment by mshanemc** [comment]
the CLI can't do that because the API doesn't support it.  Each bulk api request is for a single object and each bulk job is an independent transaction.

So there's no good way to do what you want without changing the API (there's always IdeaExchange for that)
